Automattic/jetpack
# Tiled Gallery Block: remove Plans dependency for VIP Support

Follow-up to #13293

## Proposed changes:

Instead of relying on a WordPress.com API request to get the site's plan, and thus to figure out whether the site is on VIP or not, let's rely on the site's constants. That will save us an external request, and will allow to more easily extract the Tiled Gallery block in the future if necessary.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

> [!NOTE]
> This would need to be tested on the VIP platform.

1. Go to Posts > Add New
2. Add a Tiled Gallery block
3. View the post on the frontend, ensure the images are properly displayed and do not use the Photon subdomain `i*.wp.com`.
